lib/avromatic/io/datum_reader.rb

```diff
@@ -1,86 +1,37 @@
 # frozen_string_literal: true
 
-# rubocop:disable Style/WhenThen
```

Comment:
Can you remove the corresponding enable at the bottom of the file? `rubocop:enable Style/WhenThen`

Reply:
Weird. This was removed with a `rubocop -a` but I guess that didn't remove the corresponding `rubocop:enable`.